.github/workflows/build.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build dev images
+
+on:
+  push:
+    branches:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set buildx alias
+        run: docker buildx install
+
+      - name: Install docker pushrm
+        run: |
+          sudo wget https://github.com/christian-korneck/docker-pushrm/releases/download/v1.9.0/docker-pushrm_linux_amd64 -O /usr/libexec/docker/cli-plugins/docker-pushrm
+          sudo chmod +x /usr/libexec/docker/cli-plugins/docker-pushrm
+          docker pushrm --help
```

Comment:
Does this work with github container registry?

I would be fine with leaving this out for the example images.

Reply:
Seems like it's not used in the script.